### PR TITLE
GCI-1060: Content format tweaks

### PR DESCRIPTION
**Pull request by VaughanJackson123**
* Tweaks to the format of content provided for certified copy order confirmation emails.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java [hunk 1]
```diff
@@ -58,13 +63,24 @@ default void mapCertifiedCopyItems(final OrderData order,
...
         return certifiedDocuments;
     }
+
+    /**
+     * Reformats a date filed string such as "2009-08-23" as "23 Aug 2009".
+     * @param dateFiled the date filed as reported from the filing history
+     * @return the same date rendered for display purposes
+     */
+    @Named("reformatDateFiled")
+    default String reformatDateFiled(final String dateFiled) {
```
Comment (nabeelamirch) [inline src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java:82]: The MapperUtil has some utility methods. Should this live in there as well?

Comment (VaughanJackson123) [reply src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java:82]: It's only used in one place, and it's currently located close to its point of use.

**Review by nabeelamirch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToOrderConfirmationMapperConstants.java [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 import java.time.format.DateTimeFormatter;
 
-public class OrderDataToOrderConfirmationMapperConstants {
+class OrderDataToOrderConfirmationMapperConstants {
     /** Dictates how the payment date time is rendered in a certificate order confirmation. */
     static final DateTimeFormatter DATETIME_OF_PAYMENT_FORMATTER = DateTimeFormatter.ofPattern("dd MMMM yyyy 'at' HH:mm");
+
+    /** Dictates how the date filed date is rendered in a certified copy order confirmation. */
+    static final DateTimeFormatter DATE_FILED_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy");
```
Comment (rjadhav-ch) [inline src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToOrderConfirmationMapperConstants.java:10]: Could this be renamed to match the class it represents and not the pattern?

File: src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java [hunk 3]
```diff
@@ -58,13 +63,24 @@ default void mapCertifiedCopyItems(final OrderData order,
         List<CertifiedDocument> certifiedDocuments = new ArrayList<>();
         IntStream.range(0, filingHistoryDocuments.size()).forEach(i -> {
             CertifiedDocument certifiedDocument = new CertifiedDocument();
-            certifiedDocument.setDateFiled(filingHistoryDocuments.get(i).getFilingHistoryDate());
+            certifiedDocument.setDateFiled(reformatDateFiled(filingHistoryDocuments.get(i).getFilingHistoryDate()));
             certifiedDocument.setType(filingHistoryDocuments.get(i).getFilingHistoryType());
             certifiedDocument.setDescription(filingHistoryDocuments.get(i).getFilingHistoryDescription());
-            certifiedDocument.setFee("£" + itemCosts.get(i).getCalculatedCost());
+            certifiedDocument.setFee(itemCosts.get(i).getCalculatedCost());
```
Comment (hwilliams1) [inline src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java:69]: @VaughanJackson123 Is the currency being assumed and pre-pended on when displaying the results?

Comment (VaughanJackson123) [reply src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java:69]: Yes, as was the case with certificates too. The `£` is in the templates themselves.

**Review by hwilliams1 — APPROVED** [review]
Looks OK

File: src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java (hunk 3 above)
Comment (hwilliams1) [reply src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertifiedCopyOrderConfirmationMapper.java:69]: Approved then 👍

File: src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToOrderConfirmationMapperConstants.java (hunk 2 above)
Comment (VaughanJackson123) [reply src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToOrderConfirmationMapperConstants.java]: This change has been made.

**Pull request merged**